content/reference/cd-supporting-new-ci.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Supporting A New CI System
+expanded_url: /reference/cd/
+menu:
+  reference:
+    parent: cd
+---
+
+## Overview
+
+The Pulumi CLI already supports enhanced metadata for several popular CI/CD systems. We have tried to make it easy for contributors to add/update support for CI systems. This document walks-through how you can modify the Pulumi CLI to support your own CI system or perhaps even update an existing one.
+
+The detection of metadata in a CI environment depends on some key environment variables that a CI system injects into the build environment of their build agents. The Pulumi CLI uses those environment variables to set try and determine the values for the following properties:
```

Comment:
“to set try and determine” doesn’t sound right to me

Reply:
Ah fixed it! I must have changed my mind mid-sentence. 😁